addon/components/one-way-input.js

```diff
@@ -47,9 +47,10 @@ export default Component.extend({
   keyUp(event) { this._interpretKeyEvents(event); },
 
   _interpretKeyEvents(event) {
-    const methodName = this.KEY_EVENTS[event.keyCode];
+    let methodName = this.KEY_EVENTS[event.keyCode];
 
     if (methodName) {
+      this._sanitizedValue = null;
```

Comment:
Yup, I split out the acceptance test for key events with a separate input. That input has no `update` action, only an `onenter` action.